pkg/kubelet/kubelet.go

```diff
@@ -680,6 +680,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 	klet.containerGC = containerGC
 	klet.containerDeletor = newPodContainerDeletor(klet.containerRuntime, integer.IntMax(containerGCPolicy.MaxPerPodContainer, minDeadContainerInPod))
+	klet.sandboxDeleter = newPodSandboxDeleter(klet.containerRuntime)
```

Comment:
same `deleter` vs `deletor`. Let's make consistent one way or another

Reply:
spellchecker suggests to use `deleter` and hence used it. Is it fine to change the existing one to `deleter` ?

Reply:
I like deleter over deletor too. I wouldn't rename as part of this PR, perhaps a follow up clean up.

Reply:
My biggest concern is when code does not look consistent

Reply:
Sure, will create a follow up PR